core/src/main/scala/org/apache/spark/rdd/NewHadoopRDD.scala

```diff
@@ -230,12 +251,16 @@ class NewHadoopRDD[K, V](
     super.persist(storageLevel)
   }
 
-
-  def getConf: Configuration = confBroadcast.value.value
```

Comment:
Oh, good point. I'll certainly make it public. Re: perf implication of cloning, I think this is okay since it's behind a flag and the flag is off by default, but let me know.